### ISSUE TITLE
Question: loss for multiclass detection

### ISSUE THREAD

**Issue by abhaysjuneja** [yhenon/pytorch-retinanet#236]
Does this loss implementation work for MultiClass classification? 
I see the use of Binary Cross Entropy, and it makes me wonder how it's being used.
Is this applying the One Vs. Rest approach to calculate loss for multiple classes individually?
